pkg/dao/repositories.go

```diff
@@ -5,52 +5,53 @@ import (
 	"gorm.io/gorm"
 )
 
-type PublicRepository struct {
+// Repository internal (non-user facing) representation of a repository
+type Repository struct {
 	UUID     string
 	URL      string
 	Revision string
 }
```

Comment:
what was the reason for using a "copy" struct instead of just using the model directly?

Reply:
so that the model layer is separated from any layer using the dao layer.   While the handler layer isn't using thia dao, i kinda consider the 'external_repos' layer similar to the handler layer, it should touch the dao but not the models.

Reply:
oh, it's the touching the models here: https://github.com/content-services/content-sources-backend/blob/main/pkg/external_repos/introspect.go#L86

Reply:
ohhh, i guess List() isn't even being used then.  I'll change it to use the Dao List() call and update List() to return all repos?

Reply:
oh. ha :)

makes sense